timon/conf/config.py

```diff
@@ -117,6 +118,10 @@ def get_notifier(self, name):
         notifier = self.notifiers[name] = mk_notifier(notif_cls, **notif_cfg)
         return notifier
 
+    def get_on_db_store_plugins(self):
+        """ used in timon.state.TMonState.update_probe_state """
+        (isinstance(it, OnDbStorePlugin) for it in plugins.get_all_plugins())
```

Comment:
doc string might mention what the function is supposed to do.

Currently it seems to create a generator that iterates through all plugins and yields True if the plugin is an instance of  OnDbStorePlugin. otherwise False.

But it seems the generator is discarded before any value is yielded from it.

Just guessing, but perhaps this should be:

return (plugin for plugin in  plugins.get_all_plugins() if insinstance(plugin, OnDbStorePlugin))

Reply:
yes exact, it just popped in my mind over the weekend that I wrote this wrong -.-